test/TimedTestTests.h

```diff
@@ -85,6 +120,249 @@ BENCHMARK_TEST_GROUP(TimedTestTests, TimedTest)
+                      ThreeIterationBench.Slowest.count());
+
+    // Duration based benchmarks
+    const MultilengthSleeper::Sleep DurationTestMultiplier{2200000};
+
+    const MultilengthSleeper::Sleep Pentile1Time{1 * DurationTestMultiplier};
+    const MultilengthSleeper::Sleep Pentile2Time{3 * DurationTestMultiplier};
+    const MultilengthSleeper::Sleep Pentile3Time{5 * DurationTestMultiplier};
+    const MultilengthSleeper::Sleep Pentile4Time{7 * DurationTestMultiplier};
+    const MultilengthSleeper::Sleep Pentile5Time{9 * DurationTestMultiplier};
+
+    MultilengthSleeper PentileSleeps({Pentile1Time, Pentile2Time, Pentile3Time, Pentile4Time, Pentile5Time});
+
+    const MultilengthSleeper::Sleep PentileDelta{MicroBenchmarkResults::TimeType {3 * DurationTestMultiplier}};
+    const MultilengthSleeper::Sleep Pentile1TimeUpper{Pentile1Time > PentileDelta ?
+                                                      Pentile1Time + PentileDelta :
+                                                      PentileDelta + PentileDelta};
+    const MultilengthSleeper::Sleep Pentile1TimeLower{Pentile1Time > PentileDelta ?
+                                                      Pentile1Time - PentileDelta :
+                                                      MultilengthSleeper::Sleep{0}};
+    //const MultilengthSleeper::Sleep Pentile2TimeUpper{Pentile2Time + PentileDelta};
+    //const MultilengthSleeper::Sleep Pentile2TimeLower{Pentile2Time - PentileDelta};
+    //const MultilengthSleeper::Sleep Pentile3TimeUpper{Pentile3Time + PentileDelta};
+    //const MultilengthSleeper::Sleep Pentile3TimeLower{Pentile3Time - PentileDelta};
+    //const MultilengthSleeper::Sleep Pentile4TimeUpper{Pentile4Time + PentileDelta};
+    //const MultilengthSleeper::Sleep Pentile4TimeLower{Pentile4Time - PentileDelta};
+    //const MultilengthSleeper::Sleep Pentile5TimeUpper{Pentile5Time + PentileDelta};
+    const MultilengthSleeper::Sleep Pentile5TimeLower{Pentile5Time - PentileDelta};
+
+    const MultilengthSleeper::Sleep PentileBenchmarkDuration{25000 * DurationTestMultiplier};
+    const MultilengthSleeper::Sleep PentileTotalDelta{PentileDelta * 10};
+
+    // Some test systems are just slow and take extra time so we accept double the delta upward
+    const MultilengthSleeper::Sleep PentileExpectedTotalUpper{PentileBenchmarkDuration + (2 * PentileTotalDelta)};
+    const MultilengthSleeper::Sleep PentileExpectedTotalLower{PentileBenchmarkDuration - PentileTotalDelta};
+
+    const MicroBenchmarkResults::CountType PentileSinglePassExpectedCount
+        { MicroBenchmarkResults::CountType(Pentile1Time.count() +  Pentile2Time.count() + Pentile3Time.count() +
+            Pentile4Time.count() + Pentile5Time.count()) };
+
+    // 5 is the expected multiple, but these are often slower
+    //const MicroBenchmarkResults::CountType PentileBenchmarkExpectedCountLower
+    //    { PentileBenchmarkDuration.count() / PentileSinglePassExpectedCount * 3};
```

Comment:
More obsolete?

Reply:
Yeah, I should have removed this stuff a while ago, but I wasn't sure what it would take to make this  stable when dealing with the non-deterministic subject matter.

Reply:
Fixed.